manifests/t/Tautulli/Tautulli/2.10.4/Tautulli.Tautulli.installer.yaml

```
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Tautulli.Tautulli
+PackageVersion: 2.10.4

```

```suggestion
PackageVersion: 2.10.4.1
```
1) This must match Control Panel.

2) You will need to change `manifests/t/Tautulli/Tautulli/2.10.4` to `manifests/t/Tautulli/Tautulli/2.10.4.1` so that the validation can pass.